### ISSUE TITLE
Some package is windows only

### ISSUE THREAD

**Issue by ktomi96** [suraj-singh12/any-to-pdf#15]
### Some package alternatives should be considered
`from win32com import client  # for excel to pdf`
`from docx2pdf import convert`
`from ppt2pdf import main as ppt_processor`

#13 
following code doesnt works on linux.
You should consider other packages that dont use windows api calls

Also you should consider using dependency managment such as poetry

**Comment by suraj-singh12** [comment]
Your changes are most welcome. 
During the design of the application, only `Windows` was taken into consideration. If you can find alternate dependencies for Linux, kindly mention them. 
Will do an analysis, and implementation thereafter.

**Comment by glitchdawg** [comment]
i can implement dependency management using poetry and could also look to implement api calls that also work on linux

**Comment by suraj-singh12** [comment]
That sounds good @glitchdawg. So would you like to proceed?

**Comment by glitchdawg** [comment]
> That sounds good @glitchdawg. So would you like to proceed?

Yes but to maintain sanitary code I'll have to first implement the dependency management and implement the api calls in a later push. I'll start working on the dependency management shortly

**Comment by suraj-singh12** [comment]
> > That sounds good @glitchdawg. So would you like to proceed?
> 
> Yes but to maintain sanitary code I'll have to first implement the dependency management and implement the api calls in a later push. I'll start working on the dependency management shortly

Perfect then
I am assigning this to you.

**Comment by suryan-s** [comment]
Does the package now support Linux?

**Comment by suraj-singh12** [comment]
windows only!
